uitools/toolkitcpp/cpp/Esri/ArcGISRuntime/Toolkit/AuthenticationController.cpp

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
```suggestion
```
Be weary of the pre-commit hook ghosts. Adding blank lines where no one asked for them.

Reply:
That's exactly what happened. Thanks.